pkgs/by-name/im/imhex/package.nix

```
@@ -33,19 +35,19 @@ let
   };
 
 in
-stdenv.mkDerivation rec {
+stdenv'.mkDerivation (finalAttrs: {
   pname = "imhex";
   inherit version;
 
   src = fetchFromGitHub {
     fetchSubmodules = true;
     owner = "WerWolv";
-    repo = pname;
-    rev = "v${version}";
-    hash = "sha256-8Ehpk0TjE4itQ7D9Nx74plYwABVufuYmxfxyuSqak1c=";
+    repo = "imhex";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-6QpmFkSMQpGlEzo7BHZn20c+q8CTDUB4yO87wMU5JT4=";
   };
 
-  nativeBuildInputs = [ cmake llvm python3 perl pkg-config rsync ];
+  nativeBuildInputs = [ cmake llvmPackages.llvm python3 perl pkg-config ];
```

It is possible to skip CMake's `fixup_bundle` and manually fixup the library paths (see suggestions on the `postInstall` phase below).
```suggestion
  postPatch = lib.optionalString stdenv.isDarwin ''
    substituteInPlace cmake/modules/PostprocessBundle.cmake \
      --replace-fail "fixup_bundle" "#fixup_bundle"
  '';

  nativeBuildInputs = [ cmake llvmPackages.llvm python3 perl pkg-config ];
```